charm/lib/charms/catalogue_k8s/v2/catalogue.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Charm for providing services catalogues to bundles or sets of charms."""
+
+import logging
+from dataclasses import dataclass
+from typing import Dict, List
+
+from ops.model import Application, Relation
```

Comment:
Still ops-dependent?

Reply:
Just for typing... Mmm, we should find a way to also avoid that import